package.json

```diff
@@ -280,9 +280,7 @@
     "vscode:prepublish": "yarn run build",
     "package": "vsce package --yarn",
     "fmt:check": "prettier --check .",
-    "fmt:ocaml": "git ls-files '*.ml' '*.mli' | xargs ocamlformat -i",
```

Comment:
Why did you replace `git ls-files` with find?

Reply:
I don't know, but it seems to make more sense to use `git ls-files'. I will update this.